package spring.practice.tobyspring.exception;

/*
n회 시도 후 runtimeException을 내는 checkException
 */
public class TestExcpetion extends Exception{
}
